### PR TITLE
Continnd

### PR DESCRIPTION
**Pull request by squeezyphresh**
Added some threads!

### CODE REVIEW

File: src/Searcher.h [hunk 1]
```diff
@@ -145,6 +149,12 @@ class Searcher {
      *
      * \param[in] team the team that we belong to.
      */
+    struct ScoreSort {
```
Comment (NigoroJr) [inline src/Searcher.h]: Are you using this/going to use this? Also, Node already has `operator>` and `operator<`. And don't put it right between the documentation and the declaration...

File: src/Timer.cpp [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+float Timer::get_time() {
+    return diff;
+}
+
+int Timer::get_suggested_depth(int b, float time_allotted) {
+    int i;
+    for (i = 0; i < predicted_times[b].size(); i++) {
+        if (predicted_times[b][i] >= time_allotted) break;
```
Comment (NigoroJr) [inline src/Timer.cpp]: Please surround it with curly braces.

File: src/Searcher.cpp [hunk 3]
```diff
@@ -48,11 +48,13 @@ Searcher& Searcher::operator=(Searcher&& other) {
 
 Node Searcher::search(const DomineeringState& state,
         const unsigned depth_limit) {
+    if (move_thread.joinable()) move_thread.join();
```
Comment (NigoroJr) [inline src/Searcher.cpp]: Please surround it with curly braces.

File: src/Timer.h [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    timeval t_dummy;
+    double t_start;
+    double t_end;
+    double diff;
+    std::vector<std::vector<float>> predicted_times;
+
+    public:
+    Timer() : t_start(-1), t_end(-1), diff(-1) {}
+    Timer(std::vector<std::vector<float>> pt) : t_start(-1), t_end(-1),
+    diff(-1) {
```
Comment (NigoroJr) [inline src/Timer.h]: This is way too confusing. Use this style. https://github.com/NigoroJr/uccineering/blob/master/src/Node.cpp

**Comment by NigoroJr** [comment]
:+1: Also, remove the binary file.

File: src/Searcher.h [hunk 5]
```diff
@@ -137,14 +141,6 @@ class Searcher {
     std::vector<Node> expand(const Node& parent,
             const DomineeringState& current_state);
 
-    /**
```
Comment (NigoroJr) [inline src/Searcher.h:140]: You don't have to delete this...

Comment (squeezyphresh) [inline src/Searcher.h:140]: i don't think I did...

Comment (NigoroJr) [inline src/Searcher.h:140]: You did... https://github.com/continnd/uccineering/commit/6a92346f9223b7e1a99b5c453eb25c8fbdb5861c

**Comment by squeezyphresh** [comment]
Are you talking about ScoreSort? I don't need it. It slipped in when I was fixing a conflict.

**Comment by NigoroJr** [comment]
No, you deleted the docs too.

**Comment by squeezyphresh** [comment]
Whatever you are linking me to is not showing that I deleted any docs.

**Comment by NigoroJr** [comment]
1. Go to https://github.com/continnd/uccineering/commit/6a92346f9223b7e1a99b5c453eb25c8fbdb5861c
2. Look for the section with red background.
3. Read the first 8 lines in that section.

**Comment by squeezyphresh** [comment]
Oh, the comments. Taken care of.

**Comment by NigoroJr** [comment]
Great.

**Comment by NigoroJr** [comment]
Merged.

**Pull request merged**